src/qt/forms/diagnosticsdialog.ui

```diff
@@ -112,22 +114,15 @@
        </property>
       </widget>
      </item>
-     <item row="1" column="0">
-      <widget class="QLabel" name="boincPathLabel">
-       <property name="text">
-        <string>Verify BOINC path</string>
-       </property>
-      </widget>
-     </item>
-     <item row="3" column="0">
-      <widget class="QLabel" name="verifyCPIDHasRACLabel">
+     <item row="9" column="0">
+      <widget class="QLabel" name="verifyTCPPortLabel">
        <property name="text">
-        <string>Verify CPID has RAC</string>
+        <string>Verify TCP/IP port</string>
```

Comment:
```suggestion
        <string>Verify port for full node</string>
```

Reply:
I did something similar to your suggestion.